src/webview/mod.rs

```diff
@@ -1153,6 +1172,10 @@ impl WebviewExtWindows for WebView {
   fn set_theme(&self, theme: Theme) {
     self.webview.set_theme(theme)
   }
+
+  fn set_low_memory_usage(&self, enabled: bool) {
```

Comment:
I would suggest changing this method to take an Enum instead, to allow the possibility of other memory modes.

```suggestion
  fn set_memory_usage_level(&self, memory: MemoryLevel) {
```

Reply:
Yeah, more levels may be added in the future. And this API may be promoted to cross-platform and other WebView may have different variations. I agree with making this an enum.